comparisons/CompareGraphs.py

```diff
@@ -63,13 +63,15 @@
 legNames = inputCfg['options']['legend']['titles']
 legOpt = inputCfg['options']['legend']['options']
 legTextSize = inputCfg['options']['legend']['textsize']
+ncolumns = inputCfg['options']['legend']['ncolumns']
```

Comment:
Same comment as before, could you add all the additional arguments in this config file [config_comparison.yml](https://github.com/DmesonAnalysers/DmesonAnalysis/blob/master/comparisons/config_comparison.yml)